app/forms/add_user_email_form.rb

```diff
@@ -15,10 +15,11 @@ def user
     @user ||= User.new
   end
 
-  def submit(user, params)
+  def submit(user, params, from_select_email_flow = nil)
```

Comment:
Non-blocking: Similar observation to what I commented in Kevin's PR at https://github.com/18F/identity-idp/pull/11439#discussion_r1825706932, I think this could make more sense as a constructor keyword argument, keep `#submit` concerned with `params`.